apps/web/components/AppListCard.tsx

```diff
@@ -63,31 +63,27 @@ export default function AppListCard(props: AppListCardProps) {
   }, []);
 
   return (
-    <div className={`${highlight ? "bg-yellow-100" : ""}`}>
-      <div className="flex gap-x-3 p-4">
-        {logo ? <img className="h-10 w-10" src={logo} alt={`${title} logo`} /> : null}
-        <div className="flex grow flex-col gap-y-1 truncate">
-          <div className="flex items-center gap-x-2">
-            <h3 className="truncate text-sm font-semibold text-gray-900">{title}</h3>
-            <div className="flex items-center gap-x-2">
-              {isDefault && <Badge variant="green">{t("default")}</Badge>}
-              {isTemplate && <Badge variant="red">Template</Badge>}
-            </div>
+    <ListItem
+      heading={title}
+      subHeading={description}
+      leftNode={logo ? <img className="h-10 w-10" src={logo} alt={`${title} logo`} /> : null}
+      badgePosition="heading"
+      badges={
+        <>
+          {isDefault && <Badge variant="green">{t("default")}</Badge>}
+          {isTemplate && <Badge variant="red">Template</Badge>}
+        </>
+      }
+      actions={actions}
+      expanded={children && <div className="w-full">{children}</div>}
```

Comment:
Not sure why but it is always expanded now. 

So we always show the line even if app does not have app settings: 
<img width="894" alt="Screenshot 2023-03-02 at 10 38 00" src="https://user-images.githubusercontent.com/30310907/222475780-15bb67e6-506c-4fe3-9ae2-4365ace4c1f4.png">

Reply:
Fixing thank you :)